matlab-vnc/README.md

```diff
@@ -19,27 +19,28 @@ The `Dockerfile` in this repository builds upon the base image `jupyter/base-not
 
 To build the Docker image, follow these steps:
 
-1. Select a base container image with a MATLAB R2020b or later (64 bit Linux) installation. If you need to create one, follow the steps at [Create a MATLAB Container Image](https://github.com/mathworks-ref-arch/matlab-dockerfile).
+1. Select a base container image with a MATLAB (64 bit Linux) installation. If you need to create one, follow the steps at [Create a MATLAB Container Image](https://github.com/mathworks-ref-arch/matlab-dockerfile).
 
 2. Open the `Dockerfile`, replace the name `matlab` with the name of the MATLAB image from step 1:
 
    ```
    FROM matlab AS matlab-install-stage
    ```
 
-3. If MATLAB is not installed at the location `/usr/local/MATLAB` in the image built in step 1, edit the following line to point to you MATLAB installation:
+3. MATLAB should be on the path in the image built used in step 1.
```

Comment:
Could we change "image built used" to "image used"?
Doc guidelines recommend to avoid "should" could we change this to:

MATLAB must be on the system path in the image built used in step 1.
If it is not, then edit the following line to point to your MATLAB installation: